library/Rules/Length.php

```diff
@@ -64,6 +64,10 @@ protected function extractLength($input)
         if (is_object($input)) {
             return count(get_object_vars($input));
         }
+        
+        if (is_int($input)) {
+            return mb_strlen((string)$input, mb_detect_encoding((string)$input));
```

Comment:
The use mbstring here is not necessary, since integers cannot have different encodings. Just use `strlen()`.